### PR TITLE
Docker Integration

### PR DESCRIPTION
**Pull request by dkuppitz**
Provided Docker script that allows the (parallel) execution of several build tasks within a Docker container.

VOTE: +1

### CODE REVIEW

**Comment by spmallette** [comment]
any reason this can't be targetted at tp31 and merged forward to master?

**Comment by spmallette** [comment]
actually based on the CHANGELOG contents and git tree, it looks like it should be targeted at `tp31` and not master, no?

File: docs/src/dev/developer/contributing.asciidoc [hunk 1]
```diff
@@ -90,6 +90,30 @@ mvn -Dmaven.javadoc.skip=true --projects tinkergraph-gremlin test
...
+
+The script can be found under `PROJECT_HOME/docker/build.sh`. The following tasks are currently
+supported:
+
+* run standard test suite
+* run integration tests
+* build Java docs
+* build user docs
+
+A list of command line options is provided by `docker.build.sh --help`. The container will install,
```
Comment (spmallette) [inline docs/src/dev/developer/contributing.asciidoc]: should this be "docker/build.sh --help"?

**Comment by dkuppitz** [comment]
Doh, always the same thing. I will create a new PR targeting tp31 (and you're right about the typo in the docs, I will fix that as well).

**Pull request closed without merging**